test/e2e/memtierd.test-suite/policy-age/n4c16/test03-move-idlepage/code.var.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Make sure the system has IDLEPAGE support enabled
```

Comment:
Let's keep this as is in this PR, but I think we should do some refactoring for the idlepage check and enablement. Similarly to damon-idlepage-setup in damon.source.sh, we could have a check for idlepage-setup in idlepage.source.sh, that would contain this routine.

As said, let's have a separate refactoring PR for this.